docs/rsdl/rapid-rsdl-capabilities.md

```diff
@@ -0,0 +1,337 @@
+| POST   | &#x2713;<sup>1</sup> | &#x2713;<sup>1</sup> | &#x2713;<sup>1</sup> | &#x2713;<sup>1</sup> |
+| PATCH  | &#x2713;<sup>1</sup> | &#x2713;<sup>1</sup> | &#x2713;<sup>1</sup> | &#x2713;<sup>1</sup> |
+| PUT    | &#x2713;<sup>1</sup> | &#x2713;<sup>1</sup> | &#x2713;<sup>1</sup> | &#x2713;<sup>1</sup> |
+| DELETE | |                      |                      |                      |
+
+[1] to shape the POST/PATCH/PUT response. Rarely used but supported<br/>
+[2] deleting multiple items. Rarely used but supported<br/>
+[!!TODO: check if filter segment on delete is allowed in RAPID ]
+
+## Individual Query capabilities
+
+### Filter capabilities
+
+The filter capability allows to specify which property can be used in which filter expression. There is a vast amount of possible filter expressions (see [odata abnf](https://github.com/oasis-tcs/odata-abnf/blob/main/abnf/odata-abnf-construction-rules.txt#L502)). Therefore, the filter capabilities allow to specify a few well-known but restrictive expressions or allow any expression.
+
+The format for filter capabilities is a sequence of pairs of a so called operator group and a list of property names. An operator group is constraining the form of the expression allowed in the $filter system query option.
+
+| operator group | comment                                                                                                               |
+|----------------|-----------------------------------------------------------------------------------------------------------------------|
+| eq             | `<property> eq <literal>` or <br/> `<property> in (<literal1>, <literal2>, ... ) `                                    |
+| range          | `<literal> le <property> and <property> le <literal>` <br/> or equivalent expressions with operators `ge`, `gt`, `lt` |
+| ranges         | a disjunction of `range` expressions                                                                                  |
+| prefix         | `startswith(<property>, <literal>)`                                                                                   |
+| text           | `<string op>(<property>, <literal>)`,<br/>where `<string op>` is one of `startswith`, `endswith`, `contains`         |
+| any            | any expression including expressions combined with `not`, `and`, and `or`                                             |
+
+In RSDL this 
+
+```rsdl
+path /orders {
+    GET { 
+        filter { 
+            eq     { id name description createdDate fulfillmentDate }
+            ranges { createdDate description }
+            prefix { name }
+            text   { description }                   
+        }
+
+        filter { 
+            eq     except { description }                 
+            ranges { createdDate description }
+            prefix { name }
+            text   { description }                   
+        }
```

Comment:
Really two occurrences of `filter {...}`? Or should these be two separate examples?